### ISSUE TITLE
'Decimal' name collision with QCborKnownTags (can't send any amount)

### ISSUE THREAD

**Issue by molecular** [akhavr/electrum-dash#80]
I'm on up-to-date master branch bdb39ded8fd089220688a8e940b783b9fa1afd2c

Running into an issue with my system-installed PyQt5 (version 5.12 installed here). There's a name collision on name 'Decimal', the effect of which is that nothing I enter into the amount field on the send tab get's parsed (everything returns None from BTCAmountEdit.get_amount() function.

I removed the try/except in that funciton to see the normally suppressed Exception (when entering "1" into amount field on send tab):

```
Traceback (most recent call last):
  File "/home/nick/bitcoin/electrum-dash/electrum_dash/gui/qt/main_window.py", line 728, in edit_changed
    amount = edit.get_amount()
  File "/home/nick/bitcoin/electrum-dash/electrum_dash/gui/qt/amountedit.py", line 95, in get_amount
    x = Decimal(str(self.text()))
TypeError: 'QCborKnownTags' object is not callable
```

So instead of decimal.Decimal class, "Decimal" seems to refer to QCborKnownTags class.

Similar upstream issue https://github.com/spesmilo/electrum/issues/5109 got fixed by using "no more import *": https://github.com/spesmilo/electrum/commit/026448837fb23de74fc40067c82b7caa8211cccf

Workaround is to go back to PyQ5 Version 5.11

**Comment by akhavr** [comment]
Thanks, working on this

**Comment by akhavr** [comment]
Fixed in 3.3.4 release.  

Reopen if you continue to observe this behaviour
